articles/defender-for-cloud/iac-vulnerabilities.md

```diff
@@ -188,256 +188,13 @@ Once you have set up the Microsoft Security DevOps GitHub action or Azure DevOps
-
-**Severity level**: 1
-
-#### TA-000027: Transparent Data Encryption on SQL databases should be enabled
-
-Transparent data encryption should be enabled to protect data-at-rest and meet compliance requirements.
-
-**Recommendation**: To [enable transparent data encryption](/azure/azure-sql/database/transparent-data-encryption-tde-overview?tabs=azure-portal), in the [Microsoft.Sql/servers/databases/transparentDataEncryption resource properties](/azure/templates/microsoft.sql/servers/databases/transparentdataencryption?tabs=json), add (or update) the value of the *state* property to `enabled`.
-
-**Severity level**: 3
-
-#### TA-000028: SQL servers with auditing to storage account destination should be configured with 90 days retention or higher
-
-Set the data retention for your SQL Server's auditing to storage account destination to at least 90 days.
-
-**Recommendation**: For incident investigation purposes, we recommend setting the data retention for your SQL Server's auditing to storage account destination to at least 90 days, in the [Microsoft.Sql/servers/auditingSettings resource properties](/azure/templates/microsoft.sql/2020-11-01-preview/servers/auditingsettings?tabs=json#serverblobauditingpolicyproperties-object), using the *retentionDays* property. Confirm that you're meeting the necessary retention rules for the regions in which you're operating. This is sometimes required for compliance with regulatory standards.
-
-**Severity level**: 3
-
-#### TA-000029: Azure API Management APIs should use encrypted protocols only
-
-Set the protocols property to only include HTTPS.
-
-**Recommendation**: To use encrypted protocols only, add (or update) the *protocols* property in the [Microsoft.ApiManagement/service/apis resource properties](/azure/templates/microsoft.apimanagement/service/apis?tabs=json), to only include HTTPS. Allowing any other protocols (for example, HTTP, WS) is insecure.
-
-**Severity level**: 1
+The IaC scanning tools included with Microsoft Security DevOps are [Template Analyzer](https://github.com/Azure/template-analyzer) (that itself contains [PSRule](https://aka.ms/ps-rule-azure)) and [Terrascan](https://github.com/tenable/terrascan). Template Analyzer runs rules on ARM and Bicep templates. Information about the rules in Template Analyzer, along with remediation details, can be found [here](https://github.com/Azure/template-analyzer/blob/development/docs/built-in-bpa-rules.md#built-in-rules). While Terrascan runs rules on ARM, CloudFormation, Docker, Helm, Kubernetes, Kustomize, and Terraform templates. Details about the rules covered by Terrascan can be found [here](https://runterrascan.io/docs/policies/).
 
 ## Learn more
 
-- Learn more about the [Template Best Practice Analyzer](https://github.com/Azure/template-analyzer).
+- Learn more about [Template Analyzer](https://github.com/Azure/template-analyzer).
```

Comment:
```suggestion
- Learn more about the [Template Analyzer](https://github.com/Azure/template-analyzer).
```

Reply:
I'm thinking on rejected the proposed change, TemplateAnalyzer is a piece of software, and I don't think we say "Learn more about the Excel" or "Learn more about the Edge", in fact my grammar corrector marks it as an error